optimum/intel/openvino/__init__.py

```diff
@@ -56,9 +56,9 @@
     logging.disable(logging.INFO)
     import nncf
 
-    if is_nncf_version("<", "2.19"):
+    if not is_nncf_version(">=", "2.19.0.dev0"):
         raise ImportError(
-            "NNCF version 2.19 or higher is required to use NNCF-based quantization. "
+            "NNCF version 2.19.0.dev0 or higher is required to use NNCF-based quantization. "
```

Comment:
```suggestion
            "NNCF version 2.19 or higher is required to use NNCF-based quantization. "
```

I think keeping 2.19 is ok in order not to confuse most of the users with "dev0" part.

Reply:
Updated exception message